content/docs/tasks/security/authn-policy.md

```diff
@@ -249,6 +249,62 @@ $ kubectl exec $(kubectl get pod -l app=sleep -n legacy -o jsonpath={.items..met
 200
 ```
 
+## Enable mTLS incrementally
+
+Sometimes it's difficult to enable mTLS for a service without breaking any clients. For example, a service can have clients without Envoy sidecar. Those clients
+can only send plain text traffic.
+
+To address this, Authentication Policy mTLS has two modes:
+1. `STRICT` mode. A service in this mode only accepts TLS traffic. This is the default mode.
+1. `PERMISSIVE` mode. A service in this mode accepts both TLS and plain text request on the same port.
+
+Create the `PERMISSIVE` Authentication policy for httpbin,
+```bash
+cat <<EOF | istioctl create -n foo -f -
+apiVersion: "authentication.istio.io/v1alpha1"
+kind: "Policy"
+metadata:
+  name: "example-httpbin-permissive"
+spec:
+  targets:
+  - name: httpbin
+    namespace: foo
+    peers:
+    - mtls:
+        mode: PERMISSIVE
+```
+
+Create DestinationRule to make Istio managed service send mTLS traffic,
+
+```bash
+cat <<EOF | istioctl create -n foo -f -
+apiVersion: "networking.istio.io/v1alpha3"
+kind: "DestinationRule"
+metadata:
+  name: "example-httpbin-istio-client-mtls"
+spec:
+  host: httpbin.foo.svc.cluster.local
+  trafficPolicy:
+    tls:
+      mode: ISTIO_MUTUAL
+EOF
+```
+
+Send traffic from both Istio managed service and legacy workloads:
+```command
+$ kubectl exec $(kubectl get pod -l app=sleep -n foo -o jsonpath={.items..metadata.name}) -c sleep -n legacy -- curl http://httpbin.foo:8000/ip -s -o /dev/null -w "%{http_code}\n"
+200
+$ kubectl exec $(kubectl get pod -l app=sleep -n legacy -o jsonpath={.items..metadata.name}) -c sleep -n legacy -- curl http://httpbin.foo:8000/ip -s -o /dev/null -w "%{http_code}\n"
+200
+```
+
+You should see both requests succeed.
+
+By default, when configured with `PERMISSIVE` mode, no authentication or authorization checks will be performed for the plain text traffic by default.
+We recommend to either:
+* Migrate all clients to Istio managed service, so you can change to `STRICT` mode after migration is done.
+* Use [RBAC](https://istio.io/docs/tasks/security/role-based-access-control/) to configure different paths with different authorization policies.
```

Comment:
Do we have a clear idea about how to do this? If so, maybe provide a task page. Otherwise, we should hide it for now.